### PR TITLE
Mention multi support

### PR DESCRIPTION
**Pull request by fancykingstar**

### CODE REVIEW

**Comment by fancykingstar** [comment]
Hi @sibelius @fxOne This is new PR for multi mention support funtions.

I fixed the issue for `supportWhiteSpace mention editor` and pass `mentionTrigger` for both `string` and `string[]`
Please review it and give me a feedback.
Thanks.

File: packages/docs/components/Examples/mention/CustomComponentMentionEditor/CustomComponentMentionEditor.tsx [hunk 1]
```diff
@@ -45,9 +45,12 @@ export default function CustomComponentMentionEditor(): ReactElement {
   const onOpenChange = useCallback((_open: boolean) => {
     setOpen(_open);
   }, []);
-  const onSearchChange = useCallback(({ value }: { value: string }) => {
```
Comment (fxOne) [inline packages/docs/components/Examples/mention/CustomComponentMentionEditor/CustomComponentMentionEditor.tsx:48]: Maybe we should leave the original function without the `trigger` as in the non multi mention trigger use cases the option is irrelevant and for `defaultSuggestionsFilter` optional? This comment is for all samples which are not using multi mentions.

File: packages/docs/pages/plugin/mention/index.tsx [hunk 2]
```diff
@@ -197,6 +204,14 @@ export default function Mention(): ReactElement {
             not support a multi-character mentionTrigger.
           </span>
         </div>
+        <div className={styles.param}>
```
Comment (fxOne) [inline packages/docs/pages/plugin/mention/index.tsx:207]: There are now 2 `mentionTrigger` paragraphs. I guess you can delete the old one. You also need to add the new parameter for the `defaultSuggestionsFilter` function.

File: stories/mentions-multi-triggers/src/mentions.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+const mentions = [
```
Comment (fxOne) [inline stories/mentions-multi-triggers/src/mentions.js:1]: Can you please change this file to a TypeScript file?

Comment (fancykingstar) [reply stories/mentions-multi-triggers/src/mentions.js:1]: @fxOne all `mentions` files are js file, as you can see.
image

Do you want me to change file to a typescript file for only multiTriggers?
Or do it for all?

File: packages/docs/components/Examples/mention/CustomComponentMentionEditor/CustomComponentMentionEditor.tsx (hunk 1 above)
Comment (fancykingstar) [reply packages/docs/components/Examples/mention/CustomComponentMentionEditor/CustomComponentMentionEditor.tsx:48]: Leave original function without trigger for non multi mention trigger

File: packages/docs/pages/plugin/mention/index.tsx (hunk 2 above)
Comment (fancykingstar) [reply packages/docs/pages/plugin/mention/index.tsx]: I removed old one.

File: stories/mentions-multi-triggers/src/mentions.js (hunk 3 above)
Comment (fxOne) [reply stories/mentions-multi-triggers/src/mentions.js:1]: Would be nice if you have the time to migrate them all :)

File: packages/docs/pages/plugin/mention/index.tsx (hunk 2 above)
Comment (fxOne) [reply packages/docs/pages/plugin/mention/index.tsx]: Theres also the paragraph `In addition to the plugin the module exports defaultSuggestionsFilter. As first argument it takes the search term as a String. The second argument is an array of mentions. The function returns the filter list based on substring matches.`. Would be nice if you could add the new `trigger` parameter.

**Comment by fancykingstar** [comment]
I fix issues.

**Review by fxOne — APPROVED** [review]

**Pull request merged**